test/fixtures/message/source_map_eval.js

```diff
@@ -1,10 +1,10 @@
 // Flags:  --enable-source-maps
```

Comment:
How is this a fixture? This file looks like it was intended to be run as the test itself.

Reply:
since we never cared about the exit code or anything besides stdout and stderr of running this file